### DISCUSSION TITLE
How to run only certain applications through firejail?

### DISCUSSION THREAD

**Discussion by Raj2032** [netblue30/firejail#6493] [Q&A] [answered]
I believe I have to run `firecfg` but I don't want to run it for all applications, was wondering what I should do just for `firefox` for example?

**Comment by kmk3** [comment, marked as answer]
> I believe I have to run `firecfg` but I don't want to run it for all
> applications, was wondering what I should do just for `firefox` for example?

Edit /etc/firejail/firecfg.config and leave only the programs that you want
before running `firecfg`.

Alternatively, create a wrapper script for each program.

Example for ~/bin/firefox (make sure that ~/bin is in `$PATH`):

```sh
#!/bin/sh

exec firejail /usr/bin/firefox
```

Kind of relates to:

* #5876

**Reply by rusty-snake** [reply]
> Alternatively, create a wrapper script for each program.

Or just `ln -s /usr/bin/firejail /usr/local/bin/firefox` i.e. emulate firecfg.
